Libraries/StyleSheet/processTransform.js

```diff
@@ -165,6 +165,11 @@ function _validateTransform(key, value, transformation) {
         stringifySafe(transformation),
       );
       break;
+    case 'perspective':
```

Comment:
Omitted `break;` before `default` so that `perspective` would still run through the number test.